### ISSUE TITLE
Daily note bug in determining "over due" and "due today" tasks

### ISSUE THREAD

**Issue by alicangok** [cuken/obsidian-weave#2]
Instead of `tp.date.now("YYYY-MM-DD")` for the dataview queries for "over due" and "due today" tasks, it should be handled using `<% tp.file.title %>`

Reason: A daily note is not necessarily created on the current day, e.g. one could create a daily note for next wednesday today. In this case, the task queries will show wrong tasks.

**Comment by cuken** [comment]
@alicangok I completely agree, I'll add it to the next release.

**Comment by cuken** [comment]
Should we remove the:
```
<< [[<% tp.date.now("YYYY-MM-DD", -1, tp.file.title, "YYYY-MM-DD") %>]] | [[<% tp.date.now("YYYY-MM-DD", 1, tp.file.title, "YYYY-MM-DD") %>]]>>
```
Header section in the top of the daily note?

**Comment by alicangok** [comment]
> Should we remove the: 
> Header section in the top of the daily note?

Everybody's preference is different. I removed it for myself, because I like to click from the sidebar Calendar when I need to, but I can understand if people like that header section. (I also removed a bunch of `# [[<% tp.file.title %>]]` headers from various templates.)

**Comment by cuken** [comment]
I started to include the `# [[% <tp.file.title %>]]` so that it would automatically update the H1 with the note name, if you change the file name the note name changes to reflect as it's referencing itself.

**Comment by cuken** [comment]
Fixed in dccf74098307b787b62eb043be82f71c470be3e9
